URL Detection

```diff
@@ -0,0 +1,3 @@
+NetworkCommunicationEvents  
+| where RemoteUrl == "www.microsoft.com"
```

Comment:
Consider to define this URL as a `let`:
`let partialRemoteUrlToDetect="www.microsoft.com";`
and then later in the query, use it:
`| where RemoteUrl has partialRemoteUrlToDetect`

This makes it easy to identify what this query looks for.

Reply:
Please add a comment before the query, to explain what it does.
If you chose to use `has` instead of `==`, please explain or include an example that `has` looks for whole term, while `contains` finds the expression substring.

`RemoteUrl has "microsoft"` will match `www.microsoft.com` but not `microsoftonline.com`